### ISSUE TITLE
what is  pred_boxes_deltas 

### ISSUE THREAD

**Issue by whu-loongson-wuwei** [Russell91/TensorBox#163]
delta_residual = tf.reshape(perm_truth - (pred_boxes + pred_boxes_deltas) * pred_mask,[outer_size, H['rnn_len'], 4]);
 delta_boxes_loss = (tf.reduce_sum(tf.minimum(tf.square(delta_residual), 10. ** 2)) /
outer_size * H['solver']['head_weights'][1] * 0.03)
boxes_loss = delta_boxes_loss

why  is pred_confidences  updated and pred_boxes not

**Comment by Russell91** [comment]
After rezooming, the network will try to guess how much it should shift the box from the first prediction. This allows for better accuracy. The predicted change in box coordinates is in the pred_boxes_deltas term.
